site/features.md

```diff
@@ -3,622 +3,262 @@ title: Weave Features
-are attached to the weave network before they begin execution. To use
-the proxy, run
-
-    $ eval $(weave env)
-
-and then start containers as usual.
-
-Containers started in this way that subsequently restart, either by an
-explicit `docker restart` command or by Docker restart policy, are
-re-attached to the weave network by the weave Docker API proxy.
-
-### <a name="plugin"></a>Docker network plugin
-
-Alternatively, you can use weave as a Docker plugin.  A Docker network
-named `weave` is created by `weave launch`, which you can use like this:
-
-    $ docker run --net=weave -ti ubuntu
-
-> NB: The plugin is an *alternative* to the proxy, hence one should
-> *not* run `eval $(weave env)` beforehand.
-
-For more details see the [plugin documentation](plugin.html).
-
-### <a name="addressing"></a>Address allocation
-
-Containers are automatically allocated an IP address that is unique
-across the weave network. You can see which address was allocated with
-[`weave ps`](troubleshooting.html#list-attached-containers):
-
-    host1$ weave ps a1
-    a7aee7233393 7a:44:d3:11:10:70 10.32.0.2/12
-
-Weave detects when a container has exited and releases its
-automatically allocated addresses so they can be re-used.
-
-See the [Automatic IP Address Management](ipam.html) documentation for
-further details. We also have an explanation of
-[the basics of IP addressing](ip-addresses.html)
-
-Instead of getting weave to allocate IP addresses automatically, it is
-also possible to specify an address and network explicitly, expressed
-in
-[CIDR notation](http://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation)
-\- let's see how the first example in the README would have looked:
-
-On $HOST1:
-
-    host1$ docker run -e WEAVE_CIDR=10.2.1.1/24 -ti ubuntu
-    root@7ca0f6ecf59f:/#
-
-And $HOST2:
+    host1$ eval $(weave env)
 
-    host2$ docker run -e WEAVE_CIDR=10.2.1.2/24 -ti ubuntu
-    root@04c4831fafd3:/#
+Containers started in this way that subsequently restart, either
+by an explicit `docker restart` command or by Docker restart 
+policy, are re-attached to the Weave network using the `Weave Docker API Proxy`.
 
-Then in the container on $HOST1...
+See [Running Docker Commands with the Weave Docker API](/site/weave-docker-api/using-proxy.md)
 
-    root@7ca0f6ecf59f:/# ping -c 1 -q 10.2.1.2
-    PING 10.2.1.2 (10.2.1.2): 48 data bytes
-    --- 10.2.1.2 ping statistics ---
-    1 packets transmitted, 1 packets received, 0% packet loss
-    round-trip min/avg/max/stddev = 1.048/1.048/1.048/0.000 ms
 
-Similarly, in the container on $HOST2...
+###<a name="plugin"></a>Weave Network Docker Plugin
 
-    root@04c4831fafd3:/# ping -c 1 -q 10.2.1.1
-    PING 10.2.1.1 (10.2.1.1): 48 data bytes
-    --- 10.2.1.1 ping statistics ---
-    1 packets transmitted, 1 packets received, 0% packet loss
-    round-trip min/avg/max/stddev = 1.034/1.034/1.034/0.000 ms
+Weave can also be used as a Docker plugin.  A Docker network 
```

Comment:
Maybe link to https://docs.docker.com/engine/extend/plugins_network/, and say a little bit about why this is interesting.  "... so you can use Docker's built-in network commands"